src/components/Reports/GoalCalculator/CalculatorSettings/Steps/InformationStep/InformationStepForm/InformationStepFinancialForm.tsx

```diff
@@ -178,12 +184,14 @@ export const InformationStepFinancialForm: React.FC<
                 InputProps={{
                   startAdornment: <span className="prefix">$</span>,
                   endAdornment: (
-                    <InfoIcon
-                      role="button"
-                      onClick={() =>
-                        setRightPanelContent(<Contribution403bHelperPanel />)
-                      }
-                    />
+                    <IconButton>
+                      <InfoIcon
+                        role="button"
+                        onClick={() =>
```

Comment:
MUI is complaining about this in dev console and saying to move `onClick` to the `IconButton`.